main.js

```diff
@@ -1 +1,64 @@
 import customers from './customers.js'
+
+//make first letter capital
+function capital(string){
```

Comment:
This is smart, extracting this logic into its own function. We are going to talk more about DRY and Object Oriented programming this week. https://dev.to/zaynaib/what-is-object-oriented-programming-5bl0

Reply:
You could think about which of the logic inside the big function could be abstracted into its own function.